### DISCUSSION TITLE
How do organizations work?

### DISCUSSION THREAD

**Discussion by paulgoodfield** [craftcms/console#71] [Craft Console] [answered]
Are there any docs about how the new changes in Craft ID > Craft Console work? We're an agency with many clients on Craft websites. Historically we've set up separate Craft ID accounts for each client so that they can manage their own billing details, pay for licenses etc.

I'm now trying to transfer some plugin licenses from our agency Craft ID account to one of the existing client Craft ID accounts but it's asking me which organization to transfer it to?

**Comment by brandonkelly** [comment]
You can create organizations to represent each of your clients, and invite your clients to them.

Licenses owned by organizations can be renewed using the payment method on the organization, which is set by whoever owns the organization.

So if you want your client to pay for licenses/renewals, either they can create the org themselves, or you can transfer an existing org’s ownership to them, once they’re an admin of the org. Once they are the owner, they can go to the org’s Billing page and select a payment method.

> I'm now trying to transfer some plugin licenses from our agency Craft ID account to one of the existing client Craft ID accounts but it's asking me which organization to transfer it to?

License transferring is a brand new feature, and it only works for transferring licenses between a user and their orgs, or vise-versa. We don’t support transferring between individual accounts; to do that you’d need to release the licenses and have the other person claim them.

**Comment by AugustMiller** [comment, marked as answer]
For future reference, this is covered in the [Using Organizations in Craft Console](https://craftcms.com/knowledge-base/craft-console-organizations) knowledge base article!
